.github/CODEOWNERS

```diff
@@ -43,7 +42,9 @@
 /packages/utils/telemetry-utils/                                          @microsoft/fluid-cr-telemetry
 /server/                                                                  @microsoft/fluid-cr-server
 /tools/                                                                   @microsoft/fluid-cr-tools
-/tools/markdown-magic/                                                    @microsoft/fluid-cr-tools @microsoft/fluid-cr-infra
+/tools/api-markdown-documenter                                            @microsoft/fluid-cr-docs @microsoft/fluid-cr-infra
+/tools/markdown-magic/                                                    @microsoft/fluid-cr-infra
```

Comment:
`fluid-cr-tools` doesn't apply to `/tools/markdown-magic` anymore?

Reply:
It's already included via the rule for `/tools/` - just removed the redundant declaration.